lib/interactive.dart

```diff
@@ -232,13 +238,55 @@ class Interactive {
     logger.log("${_messages.length} messages loaded");
   }
 
+  void _computePeopleProperties() {
+    _themeFrequency = {};
+    _peopleByLabel = {};
+
+    _xAxisCategories = _filters
+        .firstWhere((filter) => filter.value == _selected.metric)
+        .options;
+
+    for (var category in _xAxisCategories) {
+      _peopleByLabel[category.value] = List();
+    }
+
+    _people.forEach((people) {
+      var key;
+      switch (_selected.metric) {
+        case "age_category":
+          key = people.ageCategory;
+          SQ_IN_ROW = 12;
+          break;
+        case "gender":
+          key = people.gender;
+          SQ_IN_ROW = 24;
+          break;
+        case "idp_status":
+          key = people.idpStatus;
+          SQ_IN_ROW = 24;
+          break;
+        default:
+          logger.error("Selected metric ${_selected.metric} not found");
+      }
+      if (_peopleByLabel[key] != null) {
```

Comment:
Prefer an early exit here:
```
if (_peopleByLabel[key] == null) return;
...
```
It removes an extra indentation, and avoids accidental errors, e.g. when adding code later on, you don't need to be careful that it's in the `{}` block of the if statement or not.

Reply:
Ahh can I use a continue or return in forEach? I think I should start using for(var x in y) pattern than forEach..

Reply:
Fixed